src/nodejs/lambda-handlers/workflow-consumer.js

```diff
@@ -146,27 +146,39 @@ async function formSubmittedMethod(eventMessage) {
 }
 
 async function reviewApprovedMethod(eventMessage) {
-  const newEvent = { ...eventMessage, event_type: 'workflow_promote_step_direct' };
-  if (eventMessage.step_name === 'data_accession_request_form_review') {
-    await db.metrics.setAccessionReversion({
-      id: eventMessage.submission_id,
-      status: 'FALSE'
-    });
+  const param = { submission_id: eventMessage.submission_id, step_name: eventMessage.step_name };
+  const stepReview = await db.submission.checkCountStepReviewApproved(param);
+  if (stepReview.unapproved && parseInt(stepReview.unapproved, 10) === 0) {
```

Comment:
Please add a comment to explain how this if statement works as well as parentheses to make it more clear.